contracts.json

```diff
@@ -4,7 +4,7 @@
       "proxyAddress": "0xEdf0bef28943aE9d2ee7d84cF1AEEC85588655b7"
     },
     "BridgedPolygonNORI": {
-      "proxyAddress": "0x5D4Ccd1c007cAe21347F3ceD346389b66f7a0ccB"
+      "proxyAddress": "0xb4c857d57b7130F033159958B5af4Fa3Bd0a04c6"
```

Comment:
Can you leave this churn in the contracts.json out of the PR?  Does anyone know why this changes all the time?  Maybe the hardhat version should not be checked in at all?

Reply:
i do not know why it changes all the time esp because I just confirmed that the contract address is independent of the bytecode (just based on the sending account and the current nonce for that account).  not sure that the hardhat version would affect this either.  seems to happen when I spin up a local hardhat node and i'm often excluding `contracts.json` from my commits